src/app/code/community/Shopgate/Framework/Model/Payment/Cc/Authn.php

```diff
@@ -78,15 +78,19 @@ public function manipulateOrderWithPaymentData($order)
                     $this->getOrder()->getPayment()->setAmountAuthorized($this->getOrder()->getGrandTotal());
                     $this->getOrder()->getPayment()->setBaseAmountAuthorized($this->getOrder()->getBaseGrandTotal());
                     $this->getOrder()->getPayment()->setIsTransactionPending(true);
+                    Mage::dispatchEvent('sales_order_place_before', array('order' => $this->getOrder()));
```

Comment:
lol, you will hate me a little, but I meant here as well

Reply:
nah, I hate myself for not seeing this myself.
done